docs/reference/nomination.rst

```diff
@@ -0,0 +1,9 @@
+Nomination
+==========
+
+.. currentmodule:: graspologic.nominate
+
+Single graph spectral vertex nomination
```

Comment:
```suggestion
Spectral Vertex Nomination
```
daniel made a nomination section in reference for his PR, so put this there and delete this file